tests/test_autodoc.py

```diff
@@ -354,6 +354,16 @@ def test_resolve_links_in_text(self):
             ),
         )
 
+        self.assertEqual(
+            resolve_links_in_text(
+                "This is a regular [`link`](url)",
+                transformers,
+                small_mapping,
+                page_info,
+            ),
+            "This is a regular [`link`](url)",
```

Comment:
This test was failing, the returned value used to be "This is a regular \`link\`(url)"